src/App.js

```diff
@@ -17,45 +25,64 @@ const Wrapper = styled.section`
 `;
 
 const Text = styled.p`
-  color: white;
+  color: ${styles.colors.accentLight};
   font-size: 17px;
   font-family: "Courier new", monospace;
 `;
 
-const Title = styled(Text)` 
-  font-size: 20px; 
-  color: black;
-  background-color: papayawhip;
+const Title = styled(Text) ` 
+  font-size: 24px; 
+  color: ${styles.colors.background};
+  background-color: ${styles.colors.accent};
   padding: 5px;
-
 `;
 
 const Link = styled.a`
+  color: ${styles.colors.accentLight};
+  font-size: 17px;
+  font-family: "Courier new", monospace;
   text-decoration: none;
+  border-bottom: .5px dotted ${styles.colors.accentLight};
+  margin: 1em 0;
+`;
+
+const LinkImportant = styled(Link) `
+  font-weight: bold;
+  margin: 2em 0;
 `;
 
+const Icon = styled.svg`
+  fill: ${styles.colors.accentLight}
+  height: 20px;
+  margin: 1em 0;
+`
 
 
 class App extends Component {
 
   constructor(props) {
     super(props);
 
-    if(window.location.pathname == '/rsvp') {
+    if (window.location.pathname === '/rsvp') {
       window.location.href = 'https://uncodeferenceio.eventbrite.de';
     }
   }
 
   render() {
     return (
-      <Wrapper className="App">
-        {/*<Header />*/}
-        <Title> {"uncodeference.io"} </Title>
-        <Text> {"The web development unconference"} </Text>
-        <Text> {"Innsbruck · May 25th"} </Text>
-        <Link href="https://uncodeferenceio.eventbrite.de">
-          <Text> {"Link --> RSVP"} </Text>
-        </Link>
+      <Wrapper>
+        <Title>un<i>code</i>ference.io</Title>
```

Comment:
Is *italic* too much? 🤔

Reply:
I think it's good, because it's pretty hard to pronounce un-code-ference. This might be a help.